translation.rst

```diff
@@ -83,7 +83,7 @@ are located:
 
             <framework:config default-locale="en">
                 <framework:translator>
-                    <framework:default-path>'%kernel.project_dir%/translations'</framework:default-path>
```

Comment:
Maybe I'm missing something, but I don't understand why is it OK to remove `default-path` here because the other config formats (YAML and PHP) include that. Thanks.

Reply:
I can't tell I tried yaml configuration as shown in Symfony documention for version 4.4 and it worked, but XML format didn't, so I tried configuration for previous version of symfony and it worked, maybe it's me, maybe not, I'm just trying to save other's people time if documentation is in fact wrong about this.

Reply:
What error did you get with the previous version?